extensions/auto/src/jolie/net/auto/AutoHelper.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (C) 2014-2016 Fabrizio Montesi <famontesi@gmail.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301  USA
+ */
+package jolie.net.auto;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.URL;
+import jolie.util.Helpers;
+import org.ini4j.Ini;
+
+/**
+ *
+ * @author claudio
+ */
+public class AutoHelper {
+    
+        public static IOException buildIOException( String message )
```

Comment:
Indentation (maybe your netbeans is not applying anymore the conf file I passed once for automatic source formatting?).